src/internal.hpp

```diff
@@ -1,37 +1,44 @@
 #ifndef SENTRY_INTERNAL_HPP_INCLUDED
 #define SENTRY_INTERNAL_HPP_INCLUDED
-#include <map>
-#include <string>
+#include <errno.h>
+#include <sentry.h>
+#include <stdio.h>
 
-#include "sentry.h"
+#define SENTRY_PRINT_ARGS(message, ...)                             \
+    do {                                                            \
+        if (sentry_options_get_debug(sentry_get_options())) {       \
+            fprintf(stderr, "[sentry] " message "\n", __VA_ARGS__); \
+        }                                                           \
+    } while (0)
 
-const sentry_options_t *sentry_get_options(void);
+#define SENTRY_PRINT_DEBUG(Message) SENTRY_PRINT_ARGS("%s", Message "")
+#define SENTRY_PRINT_DEBUG_ARGS(Message, ...) \
+    SENTRY_PRINT_ARGS(Message, __VA_ARGS__)
+#define SENTRY_PRINT_ERROR(Message) SENTRY_PRINT_ARGS("%s", Message "")
+#define SENTRY_PRINT_ERROR_ARGS(Message, ...) \
```

Comment:
Nothing atm because there is no verbosity flag. Originally one was more spammy but I removed most calls but did not get to remove all yet.

Reply:
Let's add one, then, or at least allow to override these defines. It might be annoying if our SDK starts writing stuff to stderr without the ability to turn it off.

Reply:
These defines only do something if debug is enabled anyways.